docs/openapi/api-internal-pn-bff-recipient-info.yaml

```diff
@@ -63,4 +63,50 @@ paths:
           content:
             application/problem+json:
               schema:
-                $ref: 'common-refs.yaml#/components/schemas/Problem'
\ No newline at end of file
+                $ref: 'common-refs.yaml#/components/schemas/Problem'
+
+
+  '/bff/v1/pa-list':
+    get:
+      summary: Get PA list
+      description: |-
+        This operation retrieves the list of PA that use PN
+      operationId: getPAListV1
+#      security:                                      # ONLY EXTERNAL
+#        - bearerAuth: [ ]                            # ONLY EXTERNAL
+      tags:
+        - InfoRecipient
+      parameters:
+        - name: paNameFilter
+          description: Se valorizzato indica il prefisso della denominazione della PA
+          in: query
+          required: false
+          schema:
+            $ref: './info-recipient-schemas/info-recipient.yaml#/components/schemas/BffPaNameFilter'
+        - name: id
```

Comment:
On front-end side, do we use this param? if not, delete it

Reply:
removed